### PR TITLE
Refactor ScaleBitmap object to use new CreateJs class model

### PR DESCRIPTION
**Pull request by bambalaus**
Hi,
I have done a simple amended of the ScaleBitmap object in the extras folder to work with the new class system of CreateJs version 0.8. Hope I followed the expected convention

### CODE REVIEW

**Comment by lannymcnie** [comment]
Any chance you can do a pull request that modifies only the required lines? This one has huge add/remove blocks, and it is not clear what is changed.

**Comment by bambalaus** [comment]
Sorry about that the editor messed up indentation type. i did changed it, should be cleaner now

File: extras/ScaleBitmap/ScaleBitmap.js [hunk 1]
```diff
@@ -103,25 +124,14 @@ ScaleBitmap.prototype.constructor = ScaleBitmap;
 	 * @type Function
 	 * @private
 	 **/
-	p.DisplayObject_initialize = p.initialize;
+	//p.DisplayObject_initialize = p.initialize;
```
Comment (lannymcnie) [inline extras/ScaleBitmap/ScaleBitmap.js]: Any reason this is just commented out instead of removed?

**Comment by lannymcnie** [comment]
Looks much better. I provided one comment, and would be happy to accept this if you can resolve it. Cheers!

**Comment by bambalaus** [comment]
sorry for the long delay. i remote it completely now

**Comment by lannymcnie** [comment]
This has been merged in already from another pull request.

**Pull request closed without merging**